Library/Formula/svtplay-dl.rb

```diff
@@ -1,12 +1,23 @@
 class SvtplayDl < Formula
   desc "Download videos from http://svtplay.se"
   homepage "https://github.com/spaam/svtplay-dl"
-  url "https://github.com/spaam/svtplay-dl/archive/0.10.2015.08.24.tar.gz"
-  sha256 "0dcd2c6862ad0ed00a2b782f6484f005431de3229360e1797810f7adf16aedd9"
+  url "https://github.com/spaam/svtplay-dl/archive/0.20.2015.10.08.tar.gz"
+  sha256 "5b1631098a32540e92f9e99e226e0c53a64b880ecabfc6d3517fadff7ca24c4f"
 
   depends_on "rtmpdump"
 
+  resource "requests" do
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.7.0.tar.gz"
+    sha256 "398a3db6d61899d25fd4a06c6ca12051b0ce171d705decd7ed5511517b4bb93d"
+  end
+
   def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
     bin.install "svtplay-dl"
```

Comment:
Presumably we need to env script this so it can actually find the `requests` vendored?